orga/app/routes/application.js

```diff
@@ -28,11 +29,9 @@ export default class ApplicationRoute extends Route {
   async beforeModel(transition) {
     await this.session.setup();
     await this.featureToggles.load();
-    const isFranceDomain = this.currentDomain.isFranceDomain;
     const localeFromQueryParam = transition.to.queryParams.lang;
     await this.currentUser.load();
-    const userLocale = this.currentUser.prescriber?.lang;
-    await this.session.handleLocale({ isFranceDomain, localeFromQueryParam, userLocale });
+    await this.locale.detectBestLocale({ language: localeFromQueryParam, user: this.currentUser.user });
```

Comment:
Dans le cas de  pix orga, c'est as `user` mais `prescriber`:
```suggestion
    await this.locale.detectBestLocale({ language: localeFromQueryParam, user: this.currentUser.prescriber });
```